test/unit/specs/scripts/wallet.spec.js

```diff
@@ -0,0 +1,237 @@
+import {
+  createCosmosAddress,
+  sign,
+  createSignature,
+  createSignMessage,
+  generateWalletFromSeed,
+  generateSeed,
+  generateWallet,
+  createSignedTx,
+  createBroadcastBody
+} from "renderer/scripts/wallet.js"
+
+describe(`Key Generation`, () => {
```

Comment:
Minor: if you want to move the definition of replicated strings on the top of the file would be easier to read / match equal addresses etc

Reply:
we can do this together with #1891